index.html

```diff
@@ -2636,385 +2663,468 @@ <h5>Manifest Expression</h5>
 </pre>
 					</section>
 				</section>
-			</section>
-
-			<section id="extensibility">
-				<h3>Extensibility</h3>
-
-				<p>The <abbr title="information set"><a>infoset</a></abbr> is designed to provide a basic set of
-					properties for use by user agents in presenting and rendering a <a>Web Publication</a>, but MAY be
-					extended in the following ways:</p>
+			<section id="pagelist">
+				<h3>Pagelist</h3>
+				<p>
+					The <dfn>pagelist</dfn> property identifies the resource that contains the Web Publication's <a href="#wp-pagelist">pagelist</a>. It is identified by the <code>https://www.w3.org/ns/wp#pageslist</code> link relationship.
+				</p>
 
-				<ol>
-					<li>by the provision of <a href="#extensibility-linked-records">linked metadata records</a>.</li>
-					<li>through the inclusion of <a href="#extensibility-manifest-properties">additional properties in
-							the manifest</a>;</li>
-				</ol>
+				<section id="pagelist-infoset">
+					<h5>Infoset Requirements</h5>
 
-				<p>Although both methods are valid, the use of linked records to extend the infoset is RECOMMENDED.</p>
+					<p>User agents MUST compute the <code>pagelist</code> as follows:</p>
 
-				<p>This specification does not define how such additional properties are compiled, stored or exposed by
-					user agents in their internal representation of the infoset. A user agent MAY ignore some or all
-					extended properties.</p>
+					<ol>
+						<li>Identify the table of content resource: <ul>
+								<li>
+									If a resource in either the <a href="#default-reading-order">default reading order</a> or <a href="#resource-list">resource-list</a> is identified with a <code>rel</code> value including <code>https://www.w3.org/ns/wp#pageslist</code>, the corresponding <code>url</code> value identifies the pagelist resource.
+								</li>
+								<li>
+									Otherwise, the <a>primary entry page</a> is the pagelist resource.
+								</li>
+							</ul>
+						</li>
+						<li>
+							If the pagelist resource contains an HTML element with the <code>role</code>&#160;[[!html]] value <code>doc-pagelist</code>&#160;[[!dpub-aria-1.0]], the user agent MUST use that element as the pagelist.
+						</li>
+					</ol>
 
-				<section id="extensibility-linked-records">
```

Comment:
fix id

Reply:
I do not find the problem... what are you referring to?